src/Enzo/enzo_EnzoMethodPmDeposit.cpp

```diff
@@ -180,6 +208,9 @@ void EnzoMethodPmDeposit::compute ( Block * block) throw()
 
           for (int ip=0; ip<np; ip++) {
 
+	    // Stefan: Unsure about this line, doesn't match what's in the
```

Comment:
This needs to be looked at ASAP by someone who is the expert at evolving 'Particles'